README.md

```diff
@@ -58,7 +71,7 @@ In order to run this project, you need:
 ### Setup
 1. Install Ruby on Rails
 2. Clone this repo 
```

Comment:
- [ ] It is required in the README file under the  "Acknowledgements" section that an appropriate credit should be given to Gregoire Vella on Behance the author of the original design, as required by the Creative Commons license of the design. It is required to be contained inside the readme file, not as part PR description. Kindly add it here.

Reply:
@Qoosim  it is fixed  thank you